Elevel/Elevel.Api/Controllers/TestController.cs

```diff
@@ -10,46 +12,65 @@ namespace Elevel.Api.Controllers
     [Authorize]
     public class TestController : BaseApiController
     {
-
-        //[HttpPut]//To update!!!!!!!!!!!!!
-        //public async Task<IActionResult> SaveUserAnswerAsync([FromBody] UpdateTestQuestionCommand.Request request)
-        //{
-        //    await Mediator.Send(request);
-        //    return Ok();
-        //}
-
         [HttpPost]
         public async Task<IActionResult> CreateTestAsync ([FromBody] CreateTestCommand.Request request)
         {
             var result = await Mediator.Send(request);
+
             return Ok(result);
         }
 
         [Authorize(Roles = nameof(UserRole.HumanResourceManager)),HttpPost("assign")]
         public async Task<IActionResult> AssignTestAsync([FromBody] AssignTestCommand.Request request)
         {
+            var claims = User.Claims.ToList();
+
+            var userId = claims.FirstOrDefault(x => x.Type == "uid").Value;
```

Comment:
I suggest to create static class with extension to do this, because you've already write this code